scripts/remote-tmux-fuzz-marathon.sh

```
@@ -0,0 +1,82 @@
+HERE="$(cd "$(dirname "$0")" && pwd)"
+mkdir -p "$DIR"
+echo "marathon: $SEEDS seeds x $ITERS iters -> $DIR"
+
+app_pid() { pgrep -f "cmux-${CMUX_TAG}/Build.*MacOS/cmux DEV" | head -1; }
+
+relaunch_app() {
+  local pid; pid=$(app_pid)
+  [ -n "$pid" ] && kill -9 "$pid" 2>/dev/null
+  sleep 2
+  # Launch through the login shell so the app runs outside any sandbox.
+  ssh cmux-srvA "zsh -lc 'open \"$APP\"'" >/dev/null 2>&1
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)